src/core/Perspective.ts

```diff
@@ -80,28 +88,65 @@ export default class Perspective {
         throw new Error("Object is neither Link nor Expression: " + JSON.stringify(maybeLink))
     }
 
+    private renderLinksAdapter(): Promise<Ad4mPerspective> {
+        if(!this.neighbourhood || !this.neighbourhood.linkLanguage) {
+            //console.warn("Perspective.callLinksAdapter: Did not find neighbourhood or linkLanguage for neighbourhood on perspective, returning empty array")
+            return Promise.resolve(new Ad4mPerspective([]))
+        } 
+        return new Promise(async (resolve, reject) => {
+            setTimeout(() => resolve(new Ad4mPerspective([])), 20000)
```

Comment:
Hmmm perhaps, I think there was a reason that I did not throw here and instead added this graceful timeout, it might be better to throw so we can track if we get timeouts